matrix-sdk/src/main/java/org/matrix/androidsdk/fragments/MatrixMessageListFragment.java

```diff
@@ -516,22 +517,17 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
             // display a room preview
             else if (PREVIEW_MODE_READ_ONLY.equals(previewMode)) {
                 mAdapter.setIsPreviewMode(true);
-                mEventTimeLine = new EventTimeline(mSession.getDataHandler(), mRoomId);
+                mEventTimeLine = EventTimelineFactory.pastTimeline(mSession.getDataHandler(), mRoomId);
```

Comment:
It does not look like a past Timeline

Reply:
Yes but it does have the features of a past timeline = InMemory Store and fake room...
Probably need some rework then.

Reply:
ok, I've renamed the method